CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 ### Removed
 
+## 2.5.0 - 2023-05-23
+### Added
+- Added `insecureSigningMethods` to SigningRequest allowing for finer control over which simple electronic signature methods
+should be available to a signer
+```php
+$signingRequest->setInsecureSigningMethods(['draw', 'text', 'image']);
```

Comment:
These should either be enums or at least class constants people can refer to.

Reply:
On second thought, enums can't work because of the PHP version (probably), so class constants.

Reply:
added class constants